lib/realtime/channels.ex

```diff
@@ -8,32 +8,42 @@ defmodule Realtime.Channels do
 
   import Ecto.Query
 
+  def list_channels(db_conection) do
```

Comment:
nitpick, inside Postgrex everywhere used the name "conn", not "db_connection"

Reply:
good call, we should keep that pattern to make it easier to think about what a conn is